### PR TITLE
[Chore]: Changeset Actions Unification

### PR DESCRIPTION
**Pull request by camilaibs**
We currently have two actions, one to automate release pull requests and the other responsible for publishing to npm.

The first uses `changesets` to create release pull requests, but the latter doesn't.

Changeset actions can create a release pull request and publish to npm:
1. Create the pull request or append in the opened one if we push a changeset file to the main branch
2. Publish a version to npm, generate a tag for the new version, and create a new release for it in its repository when bump version pull requests are merged.

These changes propose the use of `changeset` for publishing, having only one action that automates the creation of release pull requests and publication of new versions to npm.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -101,25 +101,47 @@ techdocs-cli serve
...
+
+1. Create a new Pull Request from your branch to the main
+
+2. If you want to include your changes in the next version bump, run:
+
+```sh
+yarn changeset
+```
+
+3. Choose a package, semver version, add a summary of the changes and press ENTER
```
Comment (emmaindal) [inline README.md:116]:
```suggestion
3. Choose a package, [semver](https://semver.org/) version, add a summary of the changes and press ENTER
```

File: .github/workflows/project-board.yml [hunk 2]
```diff
@@ -9,7 +9,7 @@ on:
     types: [opened, reopened]
   pull_request:
     types: [opened, reopened]
-    branches-ignore: 'dependabot/*'
+    branches-ignore: ['dependabot/*']
```
Comment (emmaindal) [inline .github/workflows/project-board.yml:12]: ahaaaa, was that the problem!! Thanks for fixing!

File: .github/workflows/main.yml [hunk 3]
```diff
@@ -4,41 +4,31 @@ on:
...
-      - run: yarn build
-      - name: Publish
-        run: yarn workspace ${{ env.package }} publish --access public
+
+      - name: Install Dependencies
+        run: yarn install --frozen-lockfile
+
+      - name: Create Release Pull Request or Publish to npm
+        id: changesets
+        uses: changesets/action@master
```
Comment (emmaindal) [inline .github/workflows/main.yml:29]: how does this action know if it should open a new PR or publish? (just thinking as the precheck is deleted)

Comment (camilaibs) [reply .github/workflows/main.yml:29]: It pulls the current version of npm

Comment (camilaibs) [reply .github/workflows/main.yml:29]: https://github.com/atlassian/changesets/blob/main/packages/cli/src/commands/publish/publishPackages.ts#L203

**Review by emmaindal — APPROVED** [review]

File: .github/workflows/main.yml (hunk 3 above)
Comment (camilaibs) [reply .github/workflows/main.yml:29]: This is what the action does in general terms:

1. If you merge a pull request containing changeset files, the action will create a 'Version Package' PR if it does not exist, or append the merged changes to 'Version Pack' PR if it already exists.

2. The `Package Version` pull request deletes all changeset files, so when we merge it, the action will see that we have no changeset files, pulls the package versions from npm, checks for the local version hasn't been published yet, and if so, the action publish a new version, generate and submit a tag, and release in the package repository.

**Review by iamEAP — APPROVED** [review]

**Pull request merged**